pitmutationmate/src/main/kotlin/com/amos/pitmutationmate/pitmutationmate/actions/RunConfigurationAction.kt

```diff
@@ -37,5 +42,20 @@ abstract class RunConfigurationAction : AnAction() {
             var xmlListener = XMLListener(dir, editor)
             xmlListener.listen()
         }
+
+        // Update visualisation with mock results
+        val toolWindow: ToolWindow? = ToolWindowManager.getInstance(project).getToolWindow("Pitest")
+        val mutationTestToolWindowFactorySingleton = MutationTestToolWindowFactory()
+        val coverageReport: XMLParser.CoverageReport = XMLParser.CoverageReport(
```

Comment:
Maybe add a TODO comment so that we remember that this part should be updated to use the real results.